src/services/notification/notificationService.ts

```
@@ -47,11 +56,23 @@ export async function sendEmail(
     }
     return;
   }
-  if (cfg.emailProvider === "ses") {
-    logger.warn("SES provider not implemented; logging only", {
-      to: to ? "***" : undefined,
-      subject,
-    });
+  if (cfg.emailProvider === "ses" && cfg.sesAccessKeyId && cfg.sesSecretAccessKey) {
```

ESLint

[error] Replace `cfg.emailProvider·===·"ses"·&&·cfg.sesAccessKeyId·&&·cfg.sesSecretAccessKey` with `⏎····cfg.emailProvider·===·"ses"·&&⏎····cfg.sesAccessKeyId·&&⏎····cfg.sesSecretAccessKey⏎··`

(prettier/prettier)